app/src/main/java/com/reservation/camping/controller/CampsiteController.java

```diff
@@ -0,0 +1,89 @@
+package com.reservation.camping.controller;
+
+import com.reservation.camping.dto.CampsiteReservationDto;
+import com.reservation.camping.entity.AddressInfo;
+import com.reservation.camping.entity.CampsiteInfo;
+import com.reservation.camping.entity.ReservationInfo;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.HashMap;
+import java.util.Map;
+
+
+@RestController
+@RequestMapping("/booking")
+public class CampsiteController {
+    private static Map<Long, CampsiteReservationDto> testDb = new HashMap<>();    // 임시 DB
```

Comment:
```suggestion
    private final Map<Long, CampsiteReservationDto> testDb = new HashMap<>();    // 임시 DB
    
    ```suggestion
    public CampsiteController() {
        this(new HashMap<>());
    }
    
    public CampsiteController(Map<Long, CampsiteReservationDto> testDb) {
        this.testDb = testDb;
    }
```
```